src/DurableSDK/ExternalInvoker.cs

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Durable
+{
+    using System;
+    using System.Collections;
+    using System.Management.Automation;
+
+    internal class ExternalInvoker : IExternalInvoker
```

Comment:
This class just defines the interface we expect in the external SDK's orchestration invoker. It's here mostly to help document expectations and to provide some amount of type-checking.